extensions/blocks/testimonial/edit.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { TextControl } from '@wordpress/components';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from '../../utils/i18n';
```

Comment:
Just wanted to give heads up that we'll remove the i18n wrapper soon, in which point this can just be changed to `@wordpress/i18n` import. :-)

Reply:
Aight, https://github.com/Automattic/jetpack/pull/11743 is merged so this can be rewritten to:

```js
import { __ } from '@wordpress/i18n';

__( 'My testimonial', 'jetpack' ); // note textdomain!
```